x/authz/msgs.go

```diff
@@ -76,7 +75,7 @@ func (msg MsgGrant) Route() string {
 
 // GetSignBytes implements the LegacyMsg.GetSignBytes method.
 func (msg MsgGrant) GetSignBytes() []byte {
-	return sdk.MustSortJSON(legacy.Cdc.MustMarshalJSON(&msg))
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&msg))
```

Comment:
why this was wrong and why can't we use here the ugly global from `legacy`?

Reply:
Because on the ugly global from `legacy`, we didn't do RegisterConcrete/RegisterInterface https://github.com/cosmos/cosmos-sdk/pull/11224/files#diff-0a7ecdd9bf0c48da26958ebb9304dfd88cff7b1743cfbd3487ad459dc29994b3R13-R18